.vscode/launch.json

```
@@ -151,7 +151,12 @@
       "type": "chrome",
       "request": "attach",
       "name": "Attach to Electron Frontend",
-      "port": 9222
+      "port": 9222,
+      "webRoot": "${workspaceRoot}",
+      "sourceMapPathOverrides": {
+        "webpack://*": "${workspaceRoot}/*",
+        "webpack://@theia/example-electron/../*": "${workspaceRoot}/*"
+      }
```

I _think_ this can be written more generically as:

```suggestion
      "sourceMapPathOverrides": {
        "webpack:////src/*": "${workspaceFolder}/*",
        "webpack:////*": "/*"
      }
```

Which is how we have this working in Mbed Studio & Keil Studio